examples/stable_diffusion_v2/text_to_image.py

```diff
@@ -183,6 +184,15 @@ def main(args):
         sampler = PLMSSampler(model)
         sname = "plms"
 
+    # create safety checker
+    if args.check_safety:
+        if args.safety_version == "1":
+            safety_checker = SafetyChecker1(backend="ms", ckpt_path=args.clip_ckpt_path)
```

Comment:
Rename SafetyChecker1 -> SafetyCheckerV1
SafetyChecker2-> SafetyCheckerV2.

Similarly for the file name.

Reply:
done